test/e2e/etcd_multi_node_test.go

```diff
@@ -0,0 +1,441 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/gardener/etcd-druid/api/v1alpha1"
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
+	"github.com/gardener/gardener/pkg/utils/test/matchers"
+	"github.com/go-logr/logr"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/coordination/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	k8s_labels "k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("Etcd", func() {
+	Context("when multi-node is configured", func() {
+		var (
+			cl               client.Client
+			etcdName         string
+			storageContainer string
+			parentCtx        context.Context
+			provider         TestProvider
+			providers        []TestProvider
+			err              error
+		)
+
+		BeforeEach(func() {
+			parentCtx = context.Background()
+			providers, err = getProviders()
+			Expect(err).ToNot(HaveOccurred())
+			// take first provider
+			provider = providers[0]
+			cl, err = getKubernetesClient(kubeconfigPath)
```

Comment:
This is already done in suite_test.go in BeforeSuite. Do we gain something by doing this again here?